docs/spec/v1beta2/ocirepositories.md

```diff
@@ -391,13 +391,19 @@ metadata:
 spec:
   layerSelector:
     mediaType: "application/deployment.content.v1.tar+gzip"
+    operation: extract # can be 'extract' or 'copy', defaults to 'extract'
 ```
 
 If the layer selector matches more than one layer, the first layer matching the specified media type will be used.
 Note that the selected OCI layer must be
 [compressed](https://github.com/opencontainers/image-spec/blob/v1.0.2/layer.md#gzip-media-types)
 in the `tar+gzip` format.
 
+When `.spec.layerSelector.operation` is set to  `copy`,
+instead of extracting the compressed layer,
+the controller copies the tarball as-is to storage,
+thus keeping the original content unaltered.
```

Comment:
```suggestion
When `.spec.layerSelector.operation` is set to `copy`, instead of extracting the
compressed layer, the controller copies the tarball as-is to storage, thus
keeping the original content unaltered.
```